jenkins-artifacts

```diff
@@ -22,6 +22,7 @@ ARTIFACTS_MACHINE="${ARTIFACTS_USER}@${ARTIFACTS_SERVER}"
 ARTIFACT_BASE_DIR_MAIN="/data/sdt/SDT/jenkins-artifacts"
 DEFAULT_TIMEOUT=60
 SSH_OPTS="-q -o ConnectTimeout=${DEFAULT_TIMEOUT} -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o ServerAliveInterval=60"
+if [ $(uname -n | grep "cern.ch" | wc -l) -eq 0 ]; then SSH_OPTS="-J cmsbuild@lxplus.cern.ch $SSH_OPTS" ; fi
```

Comment:
@aandvalenzuela , please use `${ARTIFACTS_USER}` instead of hard-coded `cmsbuild`